### PR TITLE
checkProject helper function

### PR DESCRIPTION
**Pull request by matthewrsj**

### CODE REVIEW

**Comment by matthewrsj** [comment]
Three tests added for check_project helper function. Needs to be reviewed @tschuy @Kennric @ElijahCaine

File: tests/helpers.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var helper = require('../src/helpers');
+
+module.exports = function(expect) {
+    describe('check projects', function() {
+        it('should return true', function(done) {
+            expect(helper.check_project('gwm')).to.equal(true);
+            done();
+        });
+
+        it('should return false', function(done) {
```
Comment (iankronquist) [inline tests/helpers.js]: Can you make these more descriptive?

Comment (iankronquist) [inline tests/helpers.js]: Also, you probably shouldn't use should:
http://betterspecs.org/#should

Comment (matthewrsj) [inline tests/helpers.js]: @iankronquist do you mean something like

```it('should return false when passed invalid slug', ...``` ?

File: tests/helpers.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+var helper = require('../src/helpers');
+
+module.exports = function(expect) {
+    describe('check projects', function() {
```
Comment (tschuy) [inline tests/helpers.js]: this name should describe what the tests do do someone reading this on the command line. Perhaps give it the exact name of the helper being tested?

File: src/helpers.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+
+    check_project: function(name) {
```
Comment (tschuy) [inline src/helpers.js]: Functions in javascript should, as much as I hate to say it, camelCase.

File: tests/test.js [hunk 4]
```diff
@@ -55,3 +55,15 @@ describe('Endpoints', function() {
     require('./projects')(expect, request, base_url);
 
 });
+
+describe('Helpers', function() {
+
+    before(function(done) {
+        knex.migrate.latest().then(function() {
```
Comment (tschuy) [inline tests/test.js]: You'll need to load the fixtures in this describe as well, if you want to use the fixture's data to test the helpers.

File: tests/helpers.js (hunk 1 above)
Comment (tschuy) [inline tests/helpers.js]: ``it('returns false when passed invalid slug'``, but yes.

File: src/helpers.js (hunk 3 above)
Comment (iankronquist) [inline src/helpers.js]: We should put this in the docs and lint for it then.
https://github.com/osuosl/docs/blob/elijah/style_guides/source/development/style_guides/javascript.rst

Comment (tschuy) [inline src/helpers.js]: it's sort of in the docs: https://github.com/airbnb/javascript/blob/0aa8d97f5ba09cab807d7b5d91a8d5b2f6aad096/README.md#22.2

Comment (tschuy) [inline src/helpers.js]: I'm making a PR with it in the jscsrc. For now, this should be camelCase and I'll fix the rest.

**Comment by tschuy** [comment]
Looks good to me. Anyone else want to give input before it's implemented?

**Comment by tschuy** [comment]
Realized a few hours ago that we're going to have a pretty big problem with this -- this function will need to be asynchronous. Oops.

So the tests need to be rewritten for async.

**Comment by tschuy** [comment]
Closing since we need to rewrite these as async. I've opened #87

**Pull request closed without merging**